tests/cached.rs

```diff
@@ -898,6 +898,33 @@ async fn test_cached_sync_writes_a() {
     assert_eq!(a, c.await.unwrap());
 }
 
+#[cfg(feature = "async")]
+#[once(sync_writes = true)]
+async fn once_sync_writes_a(s: &tokio::sync::Mutex<String>) -> String {
+    let mut guard = s.lock().await;
+    let results: String = (*guard).clone().to_string();
+    *guard = "consumed".to_string();
+    results.to_string()
+}
```

Comment:
I use mutex here to verify that sync_writes really has an effect and that following calls will not execute inner.